llmproxy/tests/unit_tests/models/test_vertexai.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+from llmproxy.models.vertexai import VertexAI
+from dotenv import load_dotenv
+
+load_dotenv(".env.test")
+project_id = os.getenv("GOOGLE_PROJECT_ID")
+
+def test_invalid_project_id() -> None:
```

Comment:
This function depends on a valid application_default_credentials.json, this dependency needs to be stated or seperated

Reply:
unit tests assume we have a valid credentials json file